src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -1065,13 +1066,7 @@ public override void Cancel()
                         }
 
                         if (!_pendingCancel)
-                        { 
```

Comment:
I don't understand this change at all, especially since they seem to be completely unrelated to the focus of the PR. We don't want to be removing braces for if statements, and in this particular instance it completely upends the logic of the method.